functional/MBCS_Tests/i18n/playlist.xml

```diff
@@ -163,36 +177,6 @@ limitations under the License.
 			<version>11+</version>
 		</versions>
 	</test>
-	<test>
-		<testCaseName>MBCS_Tests_i18n_Zh_CN_aix</testCaseName>
-		<command>LANG=Zh_CN perl $(TEST_RESROOT)$(D)test.pl; \
-	$(TEST_STATUS)</command>
-		<platformRequirements>os.aix</platformRequirements>
-		<levels>
-			<level>special</level>
-		</levels>
-		<groups>
-			<group>functional</group>
-		</groups>
-		<versions>
-			<version>11+</version>
-		</versions>
-	</test>
-	<test>
-		<testCaseName>MBCS_Tests_i18n_ZH_CN_aix</testCaseName>
```

Comment:
Why we remove MBCS_Tests_i18n_Zh_CN_aix and MBCS_Tests_i18n_ZH_CN_aix?

Reply:
I do not think they are not dups. The commands are different. For example:
- LANG=zh_CN
- LANG=ZH_CN

Reply:
Aha, good catch, restored the tests.